data_structures/Queue/Circular_Queue.dart

```diff
@@ -6,7 +6,7 @@ const int MAX_SIZE = 10;
 
 class CircularQueue<T> {
   int start = -1, end = -1;
-  List<T> queue = new List<T>(MAX_SIZE);
+  List<T> queue = []..length = MAX_SIZE;
```

Comment:
I think assigning length value is not a good practice. Can we change the constructor to List.filled()